src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedObject = {};
```

Comment:
```suggestion
  const invertedRobot = {};
```
Avoid using data structures in the naming of the variables